### DISCUSSION TITLE
What should we use instead of `process.nextTick` and `setImmediate?`

### DISCUSSION THREAD

**Discussion by dancrumb** [denoland/deno#19488] [Q&A] [answered]
v0.178 removed the `node` part of the standard library.

A result of this is that we lost `nextTick` and `setImmediate`.

Did this functionality get move or renamed or is it just gone now?

**Comment by bartlomieju** [comment, marked as answer]
You can import these APIs like so:
```
import { nextTick } from "node:process";
import { setImmediate } from "node:timers";
```

**Comment by dancrumb** [comment]
Thanks!
